### PR TITLE
Test improvement - reflection rather than implement individual unit test for each property

### PR DESCRIPTION
**Pull request by anicos**
Adding a test to class `ReleaseConfiguration`. The test check whether assigning a value to a sitter returns the same value by the getter.
The test fails when key(in map) for setter is different than for getter.
The test works for setters and getters of the `ReleaseConfiguration` object and for objects inside this class such as `Git,` `Team,` etc.

Flow of test:
1. The test using reflection take all getters and setters from `ReleaseConfiguration` object.
2. Then assigning a value to each setter
3. Check if getter has expected value

### CODE REVIEW

File: src/test/groovy/org/shipkit/internal/gradle/ReleaseConfigurationGettersAndSettersTest.groovy [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+import org.shipkit.gradle.ReleaseConfiguration
+import spock.lang.Shared
+import spock.lang.Specification
+import spock.lang.Unroll
+import testutil.ReflectionUtil
+
+import java.lang.reflect.Method
+
+class ReleaseConfigurationGettersAndSettersTest extends Specification {
```
Comment (szczepiq) [inline src/test/groovy/org/shipkit/internal/gradle/ReleaseConfigurationGettersAndSettersTest.groovy]: Really good idea to keep this test separate!

**Comment by szczepiq** [comment]
Thank you for the pull request. Looking forward to merging!

File: src/test/groovy/org/shipkit/internal/gradle/ReleaseConfigurationGettersAndSettersTest.groovy (hunk 1 above)
Comment (anicos) [inline src/test/groovy/org/shipkit/internal/gradle/ReleaseConfigurationGettersAndSettersTest.groovy]: @szczepiq I changed description of PR. It is enough? Or I should add something more?

Thanks for review:)

**Comment by anicos** [comment]
The output form test look like.
`value assigned to setDryRun should be returned by isDryRun`
`value assigned to setUrl should be returned by getUrl`
`value assigned to setApiUrl should be returned by getApiUrl`
`value assigned to setRepository should be returned by getRepository`
`value assigned to setWriteAuthUser should be returned by getWriteAuthUser`
`value assigned to setReadOnlyAuthToken should be returned by getReadOnlyAuthToken`
`value assigned to setWriteAuthToken should be returned by getWriteAuthToken`
`value assigned to setLabelMapping should be returned by getLabelMapping`
`value assigned to setIgnoreCommitsContaining should be returned by getIgnoreCommitsContaining`
`value assigned to setFile should be returned by getFile`
`value assigned to setUser should be returned by getUser`
`value assigned to setEmail should be returned by getEmail`
`value assigned to setReleasableBranchRegex should be returned by getReleasableBranchRegex`
`value assigned to setTagPrefix should be returned by getTagPrefix`
`value assigned to setCommitMessagePostfix should be returned by getCommitMessagePostfix`
`value assigned to setDevelopers should be returned by getDevelopers`
`value assigned to setContributors should be returned by getContributors`
`value assigned to setPreviousReleaseVersion should be returned by getPreviousReleaseVersion`

**Comment by szczepiq** [comment]
Merging :) Thank you!

**Pull request merged**